layouts/partials/menu.html

```diff
@@ -1,11 +1,12 @@
 <ul>
-	{{ range . }}
-	<li>
+	{{ range $.menu }}
+	<li{{if $.node.HasMenuCurrent "main" . }} class="parent-active"{{end}}{{if $.node.IsMenuCurrent "main" . }} class="active"{{end}}>
```

Comment:
Shouldn't this be using `else if`:
```php
{{ if $.node.HasMenuCurrent "main" . }}
   class="parent-active"
{{ else if $.node.IsMenuCurrent "main" . }}
    class="active"
{{ end }}
```
Of course it can/should still be on a single line.